templates/nginx.conf.nj

```diff
@@ -41,7 +41,16 @@ http {
     rewrite ^/{{ page.from }}(.*)$ https://{{ page.to }}.18f.gov$1;
     {% endfor %}
 
+    rewrite ^/intake/.* https://18f.gsa.gov/contact/ redirect;
+    rewrite ^/designstandards/.* https://standards.usa.gov/ redirect;
+    rewrite ^/pages/.* https://github.com/18F/pages/ redirect;
```

Comment:
I don't think so, though we're want to keep sending /pages/whatever to pages.18f.gov